Documentation/ApiOverview/DependencyInjection/Index.rst

```diff
@@ -531,29 +531,16 @@ that uses :php:`GeneralUtility::makeInstance()` and :ref:`Extbase controllers
 What to make public
 -------------------
 
-Instances of :php:`\TYPO3\CMS\Core\SingletonInterface` and Extbase controllers
-are automatically marked as public. Some further classes must be marked as
-public, too. As the Symfony documentation "Public and private services" puts
-it:
-
-.. note::
-
-   A service can be marked as private if you do not want to call
-   it directly from your code.
-
-Symfony: How to Create Service Aliases and Mark Services as Private: SymfonyPrivate_
+Every class that is instantiated using :php:`GeneralUtility::makeInstance()`
+**and** requires dependency injection must be marked as public. The same goes
+for instantiation via :php:`GeneralUtility::makeInstance()` using constructor
+arguments.
 
-..  _SymfonyPrivate: https://symfony.com/doc/current/service_container/alias_private.html
+Any other class which requires dependency injection and is retrieved by
+dependency injection itself can be private.
 
-Direct access includes instantiation via :php:`GeneralUtility::makeInstance()`
-with constructor arguments.
-
-This means every class that is directly retrieved using
-:php:`GeneralUtility::makeInstance()` *and* requires dependency injection
-**must** be marked as public. Any other class which requires dependency injection
-and is retrieved by dependency injection itself can be private. Instances of
-:php:`\TYPO3\CMS\Core\SingletonInterface` and Extbase controllers are
-automatically marked as public because they are retrieved using
+Instances of :php:`\TYPO3\CMS\Core\SingletonInterface` and Extbase controllers
```

Comment:
I have read this sentence three times and it looks like instanciating classes with GU::makeInstance will "automatically" mark classes as public. Sure, thats wrong. I would prefer to write that in more detail.

Reply:
lol, interesting that you actually found a 2nd way to read this.

You: fetch via `GU::mI` -> `public: true`
Actual: `public: true` to allow `GU::mI`.

I'll have a look. ;-)

Reply:
Rephrased now, can you have a look/read?